google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -31,6 +32,61 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def to_proto_value(value):
+    """translates a Python value to a google.protobuf.Value.
+
+  Args:
+  value: The Python value to be translated.
+
+  Returns:
+  The translated google.protobuf.Value.
+  """
+    # possible Python types (this is a Python3 module):
+    # https://simplejson.readthedocs.io/en/latest/#encoders-and-decoders
+    #   JSON              Python 2        Python 3
+    # object              dict            dict
+    # array               list            list
+    # string              unicode         str
+    # number (int)        int, long       int
+    # number (real)       float           float
+    # true                True            True
+    # false               False           False
+    # null                None            None
+    if value is None:
+        # translate null to an empty value.
+        return struct_pb2.Value(null_value=struct_pb2.NullValue.NULL_VALUE), None
+    elif isinstance(value, bool):
+        # This check needs to happen before isinstance(value, int),
+        # isinstance(value, int) returns True when value is bool.
+        return struct_pb2.Value(bool_value=value), None
+    if isinstance(value, six.integer_types) or isinstance(value, float):
```

Comment:
if or elif should work for all of these?

Reply:
Sorry I don't quite understand, do you mean if these if-elif statements (from line 55 to 85) covers all supported data types? Yes.

Reply:
sorry poor communication. just an observation, on this line, you started a new if clause. this if can be elif OR all of the elif can be if. no change necessary.